modules/ibi-create-config-iso.adoc

```diff
@@ -211,7 +213,7 @@ spec:
 <3> Specify the name of the `ClusterDeployment` resource that you want to use for the image-based installation of the target host.
 <4> Specify the hostname for the cluster.
 <5> Specify the name of the `ClusterImageSet` resource you used to define the container release images to use for deployment.
-<6> Specify the public CIDR (Classless Inter-Domain Routing) of the external network.
+<6> Specify the public CIDR (Classless Inter-Domain Routing) of the external network. For dual-stack networking, you can specify both IPv4 and IPv6 CIDRs using a list format. For a cluster with the IPv4 family as the primary address family, specify the IPv4 setting first. For a cluster with the IPv6 family as the primary address family, specify the IPv6 setting first.
 <7> Optional: Specify a proxy to use for the cluster deployment.
 +
 [IMPORTANT]
```

Comment:
Dan - see this note relating to proxy networking, just checking if there's anything to update here or if it's ok?